app/EvenCountColorCode.js

```diff
@@ -0,0 +1,44 @@
+import { ColorPair } from "./colorPair.js";
+export class EvenCountColorCode {
+    MajorColorNames = [
+        "WHITE", "RED", "BLACK", "YELLOW", "VIOLET"
+    ];
+    MinorColorNames = [
+        "BLUE", "ORANGE", "GREEN", "BROWN", "SLATE"
+    ];
+    COLOR_PAIRS = {}
+    COLOR_PAIRS_KEY_VALUE_REVERSED = {}
+
+    constructor(){
+        let minorColorSize = this.MinorColorNames.length
+        let majorColorSize = this.MajorColorNames.length
+        for (let i = 0; i < majorColorSize; i++){
+            for (let j = 0; j < minorColorSize; j++){
+                this.COLOR_PAIRS[(i*majorColorSize)+(j + 1)] = [this.MajorColorNames[i],this.MinorColorNames[j]]
+                this.COLOR_PAIRS_KEY_VALUE_REVERSED[[this.MajorColorNames[i],this.MinorColorNames[j]]] = (i*majorColorSize)+(j + 1)
+            }
+        }
+    }
+    getManual = ()=>{
+        let manualTemplate = "COLOR PAIR - MANUAL\n";
+        Object.keys(this.COLOR_PAIRS).forEach(key => {
+            manualTemplate += `Pair No: ${key} => [MajorColor: ${this.COLOR_PAIRS[key][0]}, MinorColor: ${this.COLOR_PAIRS[key][1]}]\n`
+        })
+        return manualTemplate
```

Comment:
Good approach to separate the formatting from the console log. But how do you prove this function?
Can you think of a test for this function? And can you tune the test for customer satisfaction?
1. Think what a customer would want, when they print the manual
2. Make the test such that, if it passes then we're sure the customer would be satisfied.

Reply:
I have added description, usage, 25-color-codes, reference to the manual.
And I have also added a test `TestManual()`  in the TestFunction.js file